docs/usage/configuration-options.md

```diff
@@ -1652,6 +1652,31 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### replacementName
```

Comment:
@JamieMagee The last two issues are not linked to be closed with this PR. I think this is because of a syntax error. I've checked the GitHub docs [^gh-docs], and I think this should work properly:

```markdown
Closes #2223, closes #5535, closes #3339
```

Relevant table from the GitHub Docs:

> Linked issue | Syntax | Example
> --------------- | ------ | ------
> Issue in the same repository | *KEYWORD* #*ISSUE-NUMBER* | `Closes #10`
> Issue in a different repository | *KEYWORD* *OWNER*/*REPOSITORY*#*ISSUE-NUMBER* | `Fixes octo-org/octo-repo#100`
> Multiple issues | Use full syntax for each issue | `Resolves #10, resolves #123, resolves octo-org/octo-repo#100`

[^gh-docs]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Reply:
Thanks! I've update the body of the PR.